corehq/apps/saved_reports/models.py

```diff
@@ -203,7 +204,8 @@ def _dispatcher(self):
             CustomProjectReportDispatcher,
             EnterpriseReportDispatcher,
             ReleaseManagementReportDispatcher,
-            CaseManagementMapDispatcher
+            CaseManagementMapDispatcher,
+            DomainReportDispatcher,
```

Comment:
Do we know why this was not previously included here? Like was it an accident, or was it specifically not included for some other reason? I'm not sure how to evaluate if adding this here has other unexpected consequences.

Reply:
This seems to me like an accident, or some timing window where one class existed without the other. `_dispatcher` is only called in 4 locations: 
- adding a 'permissions_check' property to view_kwargs (which only seems to be used in emailing the report)
- creating a URL to the report
- getting the report
- getting the report as an email

This dispatcher is only used by the Data Forwarding Records Report and the Inbound API Request Logs Report (which is gated behind a solutions feature flag). Neither of these reports appears to have an option to download the report, so generating the report as email, from above, shouldn't be used. 

This seems low risk to me. The main side effect is that saving filters will cause a report accessed via Project Settings to appear in Saved Reports, which I'm not sure is even inconsistent (the user did click the 'save' button on a report).